.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  # schedule:
+  #   - cron: "0 23 * * 6"
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - "ubuntu-latest"
+          # - "macos-latest"
+          # - "windows-latest"
+        node_version:
+          - "18"
```

Comment:
Prettier v3 will support Node.js >= 14, so we should test parse-srcset on 14, 16, 18?

Reply:
`node:test` module only available on 18

Reply:
Hmm, so I think we should use other lightweight testing library (e.g.https://github.com/lukeed/uvu) instead of `node:test`.

Reply:
I don't think it's nessary.

Reply:
Okay, I don't have string opinion. Let's consider this when we actually something happend.